src/graphics/resources/buffer.rs

```diff
@@ -51,7 +63,7 @@ pub fn create_transfer_buffer(texture: &Texture, device: &Device) -> Buffer {
     device.create_buffer(&wgpu::BufferDescriptor {
         label: label!("TransferBuffer"),
         size: texture.bytes_stride as u64 * texture.size.height as u64,
-        usage: BufferUsages::COPY_DST | BufferUsages::MAP_READ,
+        usage: BufferUsages::MAP_READ | BufferUsages::COPY_DST,
```

Comment:
Really? :)

Reply:
Haha, yes I'm like that sometimes. It's stupid but BufferUsages::MAP_READ is less common and I feel more important. I tried to sort the flags by relevancy. I changed this when I had to set the flags in `init_image_vertex_buffer()`.